.github/workflows/unit_tests.yml

```diff
@@ -0,0 +1,78 @@
+name: unit tests
+
+on: [push, pull_request]
+
+env:
+  NODE_VERSION: 10
+
+jobs:
+  python-unit:
+    name: Python unit tests
+    runs-on: ubuntu-latest
+    # We use a service container to avoid needing to set up a local copy of
+    # mysql on the test runner instance. This syntax is similar to the spec of
+    # a docker-compose file. For more, see:
+    # https://docs.github.com/en/free-pro-team@latest/actions/guides/about-service-containers
+    services:
+      db:
+        image: mysql:5.6  # oldest available on docker hub; QA/Prod use 5.5
+        env:
+          MYSQL_DATABASE: cdhweb
+          MYSQL_ROOT_PASSWORD: cdhweb
```

Comment:
I presume these have to match our test settings? Might be nice to set them once for both, but seems fine to hardcode here for now.

Reply:
They do, yeah - I added a note in the testsettings. Probably if we wanted we could do something slightly smart with parsing the `testsettings.py` file (maybe just `awk` or `sed`?) and set them as envvars earlier in the build, in a similar manner to the `.python-version` trick

Reply:
Environment variables seems simplest. We could just adjust `testsettings.py` to set configurations from environment variables (simpler than parsing and rewriting the file!).

Reply:
Makes sense!